pkg/reconciler/taskrun/metrics.go

```diff
@@ -58,6 +58,10 @@ var (
 	podLatency = stats.Float64("taskruns_pod_latency",
 		"scheduling latency for the taskruns pods",
 		stats.UnitMilliseconds)
+
+	cloudEvents = stats.Float64("cloudevent_count",
```

Comment:
Can this be `int64` instead of `float64`? we could casting it to int [below](https://github.com/tektoncd/pipeline/pull/2720/files#diff-27bade07d6f9ea090b1a933d80905ef3R377)

Reply:
Count being `float` type looks weird 🤔

Reply:
Thank you for the review. Fixed.